lms/djangoapps/discussion_api/api.py

```diff
@@ -696,7 +696,11 @@ def get_thread(request, thread_id):
         thread_id: The id for the thread to retrieve
 
     """
-    cc_thread, context = _get_thread_and_context(request, thread_id)
+    cc_thread, context = _get_thread_and_context(
+        request,
+        thread_id,
+        retrieve_kwargs={"user_id": unicode(request.user.id)}
+    )
```

Comment:
Is there a use case for letting the user pass another user's user.id?  If not, I'd rather see the ruby code getting the id from the authenticated user on that side.